8Knot/assets/color.css

```
@@ -1,412 +1,578 @@
-  font-size: 14px;
-  font-weight: 500;
-  color: var(--color-white);
-  margin-bottom: 0;
-  margin-right: 0.5rem;
-  margin-bottom: 0;
+    font-size: 14px;
+    font-weight: 500;
+    color: var(--color-white);
+    margin-bottom: 0;
+    margin-right: 0.5rem;
+    margin-bottom: 0;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

margin-bottom is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)